SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift

```diff
@@ -165,4 +169,41 @@ class AppCenterViewController : NSViewController, NSTextFieldDelegate, CLLocatio
         signInController.action = currentAction
     }
   }
+  
+  @IBAction func setLogURL(_ sender: NSButton) {
+    let alert: NSAlert = NSAlert()
+    alert.messageText = "Log URL"
+    alert.addButton(withTitle: "Reset")
+    alert.addButton(withTitle: "Save")
+    alert.addButton(withTitle: "Cancel")
+    let scrollView: NSScrollView = NSScrollView(frame: NSRect(x: 0, y: 0, width: 300, height: 40))
+    let textView: NSTextView = NSTextView(frame: NSRect(x: 0, y: 0, width: 290, height: 40))
+    textView.string = UserDefaults.standard.string(forKey: kMSLogUrl) ?? prodLogUrl()
+    scrollView.documentView = textView
+    scrollView.hasVerticalScroller = true
+    scrollView.contentView.scroll(NSPoint(x: 0, y: textView.frame.size.height))
+    alert.accessoryView = scrollView
+    alert.alertStyle = NSWarningAlertStyle
+    switch(alert.runModal()) {
+    case NSAlertFirstButtonReturn:
+      UserDefaults.standard.removeObject(forKey: kMSLogUrl)
+      appCenter.setLogUrl(prodLogUrl())
+      break
+    case NSAlertSecondButtonReturn:
+      let text = textView.string ?? ""
+      let logUrl = !text.isEmpty ? text : nil
```

Comment:
Are you sure setting log url to `nil` is ok? I would not set anything and roll back to previous url in such case.

Reply:
Ah nvm just saw the second PR :)